collect_app/src/main/java/org/odk/collect/android/widgets/ArbitraryFileWidget.java

```diff
@@ -171,7 +175,10 @@ private void setUpLayout() {
         widgetLayout.addView(chooseFileButton);
         widgetLayout.addView(answerLayout);
 
-        addAnswerView(widgetLayout);
+        Resources resources = this.getResources();
+        int marginStandard = dpFromPx(getContext(), resources.getDimensionPixelSize(R.dimen.margin_standard));
+        int margin = marginStandard - FIELD_HORIZONTAL_MARGIN_MODIFIER;
```

Comment:
This `FIELD_HORIZONTAL_MARGIN_MODIFIER` should only really be used on the `StringWidget` as it will make the margin smaller than it should be. Could you make sure to only modify the standard margin on the widgets that you feel need it? If it's not just the `StringWidget` that needs it we should move the constant somewhere else.